### PR TITLE
Feature/reports on alpine

### PR DESCRIPTION
**Pull request by heljan**
Replacing current base image from **ubuntu:16.04** to **httpd:alpine**. The change caused also some other minor changes like configuration paths.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Not released yet
 
+# 0.12.0
```
Comment (Skejven) [inline CHANGELOG.md]: That's would be probably the next version, but this is still `Not released yet`. Please remove that version.

File: report/Dockerfile [hunk 2]
```diff
@@ -16,38 +16,45 @@
 # limitations under the License.
 #
 
-FROM ubuntu:16.04
-LABEL maintainer="Maciej Laskowski <skejven@gmail.com>"
+FROM httpd:alpine
+LABEL maintainer="Jan Helak <https://github.com/heljan>"
```
Comment (Skejven) [inline report/Dockerfile]: Could you please update to `maintainers`?
```suggestion
LABEL maintainers="Jan Helak <https://github.com/heljan>, Maciej Laskowski <skejven@gmail.com>"
```

File: CHANGELOG.md (hunk 1 above)
Comment (heljan) [reply CHANGELOG.md]: corrected

File: report/Dockerfile (hunk 2 above)
Comment (heljan) [reply report/Dockerfile]: corrected

File: report/Dockerfile [hunk 3]
```diff
@@ -16,38 +16,46 @@
...
 COPY suite-generator-beta.zip /tmp
-RUN unzip -o /tmp/suite-generator-beta.zip -d /var/www/html/suite-generator && rm /tmp/suite-generator-beta.zip
+RUN mkdir -p ${APACHE_WWW_DIR}/suite-generator 
 
-EXPOSE 80
+RUN chown -R ${APACHE_USER}:${APACHE_USER} ${APACHE_WWW_DIR}
 
-CMD ["apachectl", "-D", "FOREGROUND"]
\ No newline at end of file
+RUN httpd -k restart
```
Comment (Skejven) [inline report/Dockerfile]: Why do we need to restart `httpd` here?
Apache server wasn't running till this moment.

Comment (Skejven) [reply report/Dockerfile]: I will remove this entry in the `master` branch.

**Pull request merged**